### PR TITLE
Windows UI fallbacks when QT_QUICK_BACKEND="software"

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
## Description

 Adds fallback treatments for UI components that require VPNOpacityMask {} and become invisible when `QT_QUICK_BACKEND="software"`. Does not restore ring animation. 

🚧 Beware placeholder gunk in commandui.cpp. Once #3398 is merged this can be replaced with some real. 🚧

<table>
<tr>
<td> 
Screen Shot 2022-04-29 at 12 23 22 PM
</td>

<td> 
Screen Shot 2022-04-29 at 12 23 28 PM
</td>
</tr>
</table>

## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: nebula/ui/components/VPNMainImage.qml [hunk 1]
```diff
@@ -56,8 +55,8 @@ Rectangle {
 
         },
         State {
-            name: "stateDisconnecting"
```
Comment (bakulf) [inline nebula/ui/components/VPNMainImage.qml:59]: this seems to be an mistake :)

**Review by bakulf — APPROVED** [review]
Restore the 'StateDisconnecting'. Maybe you can push this to the windows 'gg' branch?

File: nebula/ui/components/VPNMainImage.qml (hunk 1 above)
Comment (lesleyjanenorton) [reply nebula/ui/components/VPNMainImage.qml:59]: definitely a mistake. Will fix this up and push to the gg branch.

**Comment by lesleyjanenorton** [comment]
Closing and pushing this to the 'gg' branch

**Pull request closed without merging**